GetIntoTeachingApi/Models/ExistingCandidateRequest.cs

```diff
@@ -8,13 +7,9 @@ public class ExistingCandidateRequest
     {
         private static readonly int MinimumAdditionalAttributeMatches = 2;
 
-        [SwaggerSchema("First name")]

```

Comment:
are annotations no longer needed for swagger?

Reply:
They are, but I've decided to only put them in when they actually add some value... this seemed a little pointless on reflection 🙄